example/index.html

```diff
@@ -39,6 +55,46 @@ <h2>Scope</h2>
       <li><cite id="bib-ietf-rfc-8259">IETF RFC 8259</cite>, The JavaScript
       Object Notation (JSON) Data Interchange Format, T. Bray, <a>http://www.ietf.org/rfc/rfc8259.txt</a></li>
     </ul>
+
+  </section>
+  <!-- End of Normative references -->
+
+  <!-- Start of Terms and definitions - Do not remove this section -->
+  <section id="sec-terms-and-definitions">
+
+    <!-- Use the `p` element to list external documents where Terms are defined (use same logic as referenced docs in the body of the document as example shows). Add `dt/dd` sub element pairs as needed for Terms to be defined internally in this document. Remove either the `p` if no external Terms, or `dl` if no internally defined Terms. If no Terms needed to be defined at all, remove the both `p` and `dl` elements. Pre text for this section is auto generated based on what elements are present. -->
+
+    <p hidden id="terms-ext-defs"><a href="#bib-ietf-rfc-8259"></a>, <a href="#bib-ietf-rfc-7405"></a>,</p>
+
+    <dl id="terms-int-defs">
+      <dt><dfn>Group</dfn></dt>
+      <dd>
+        <p>is the group that authored the document (e.g., TC name, “31FS”)</p>
```

Comment:
is the `p` necessary in the `dd`?

Reply:
Yes to have the spacing correct.

Reply:
Updated via https://github.com/SMPTE/html-pub/pull/27/commits/1d1c2d8b590b01bba6c7d31eb9773b5271602c91